docs/user/openstack/deploy_baremetal_workers.md

```diff
@@ -0,0 +1,189 @@
+# Deploying OpenShift bare-metal workers on OpenStack cloud provider
+TBD introduction
+
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [Considerations whe deploying bare-metal workers](#conciderations-whe-deploying-bare-metal-workers)
+- [Deploying cluster with BM workers on tenant network deployed by the installer](#deploying-cluster-with-bm-workers-on-tenant-network-deployed-by-the-installer)
+- [Deploying cluster with BM workers on preexisting network](#deploying-cluster-with-bm-workers-on-preexisting-network)
+- [Deploy cluster with vm workers only and add BM workers after](#deploy-cluster-with-vm-workers-only-and-add-bm-workers-after)
+
+
+## Common prerequisites
+* [Ironic bare metal service][2] is enabled and accessible through the [OpenStack Compute API][3].
+* Bare-metal machines are available as an [OpenStack flavor][1]
+* At the time bare-metal machines are booted the image used to boot them is available from the [Glance OpenStack image service][4].
+If given a url, the installer will fetch that image from the location specified and upload it to Glance. See relevant documentation[5] and clusterOSImage[6]. 
+* [Ironic bare metal service][2] can listen for and PXE-boot machines in tenant networks
```

Comment:
This is only a prerequisite when deploying BMs in the tenant network, so only for case 1 if I'm not mistaken